### ISSUE TITLE
[Blueprint] Dragon Ball Super

### ISSUE THREAD

**Issue by archieyates** [TitleCardMaker/Blueprints#565]
### Series Name

Dragon Ball Super

### Series Year

2015

### Series Database IDs

imdb:tt4644488,tmdb:62715,tvdb:295068

### Creator Username

Reicha7

### Blueprint Description

Textless Title Cards using the show's own cards

### Blueprint

```json
{
  "series": {
    "card_type": "textless",
    "template_ids": [],
    "auto_split_title": true,
    "source_files": []
  }
}
```

### Preview Title Cards

![preview](https://github.com/TitleCardMaker/Blueprints/assets/9253709/1c48e695-6d15-487a-be83-d195c670621a)
![preview5](https://github.com/TitleCardMaker/Blueprints/assets/9253709/dc387025-b0c7-48f9-8bf8-9b24d3d3d2cd)
![preview4](https://github.com/TitleCardMaker/Blueprints/assets/9253709/e653a1de-72c6-41c2-bc80-be43dcb4b7f5)
![preview3](https://github.com/TitleCardMaker/Blueprints/assets/9253709/e3696553-aac4-4d29-8359-be15f0404591)
![preview2](https://github.com/TitleCardMaker/Blueprints/assets/9253709/be244d94-a47c-4e5d-84de-37a7097c5371)

**Comment by archieyates** [comment]
Link to the source files since the zip was too big: https://drive.google.com/file/d/11orTBLK7NjsYzAnVB2LrrKeS3YVoQNko/view?usp=drive_link

**Comment by CollinHeist** [comment]
Accidentally marked this as created. I haven't added this yet because I am trying to think of a better option than loading this repo up with lots of source images (since that really slows down the automated actions and will eventually not work as the repo size limit is reached)

**Comment by archieyates** [comment]
Yeah I understand that. Could source files turn into a hyperlink of some sort? Although I guess that adds potential risk though as the uploader could change what exists at the link location
